src/Files.App/ViewModels/ItemViewModel.cs

```diff
@@ -407,6 +407,8 @@ public ItemViewModel(FolderSettingsViewModel folderSettingsViewModel)
             UserSettingsService.OnSettingChangedEvent += UserSettingsService_OnSettingChangedEvent;
             FileTagsSettingsService.OnSettingImportedEvent += FileTagsSettingsService_OnSettingImportedEvent;
             FolderSizeProvider.SizeChanged += FolderSizeProvider_SizeChanged;
+
+            _ = InitializeConnectionAsync(); // fire and forget
```

Comment:
I don't think it should belong in the constructor

Reply:
Copied from YourHomeViewModel. Moved outside the constructor now.